qa/tasks/vstart_runner.py

```diff
@@ -308,10 +308,22 @@ def run(self, args, check_status=True, wait=True,
 
         return proc
 
-    def sh(self, command, log_limit=1024, cwd=None, env=None):
+    # XXX: for compatibility keep this method same teuthology.orchestra.remote.sh
+    def sh(self, script, **kwargs):
+        """
+        Shortcut for run method.
+
+        Usage:
+            my_name = remote.sh('whoami')
+            remote_date = remote.sh('date')
+        """
+        if 'stdout' not in kwargs:
+            kwargs['stdout'] = BytesIO()
```

Comment:
stdout is StringIO instance instead of BytesIO at master - https://github.com/ceph/ceph/blame/master/qa/tasks/vstart_runner.py#L434. So, I think we should backport this commit too along - https://github.com/ceph/ceph/commit/624944503e7c6aa246331f4d9f291cf714296380.

I am not aware of how it works when we have 2 PRs with related changes that needs to be backported -- do we backport this other commit in this PR itself or do I need to request/create a separate backport?

Reply:
@smithfarm While checking [this issue](https://github.com/ceph/teuthology/pull/1459), I think it's better cherry-pick 624944503e7c6aa246331f4d9f291cf714296380 too in this PR.

Reply:
@rishabh-d-dave @joscollin I could agree the 6249445 is related, but from the comment in the code:
```
# XXX: for compatibility keep this method same teuthology.orchestra.remote.sh
```
But the 6249445 does not make this method the same as _remote.sh_ because it changes the default behavior of the method. Maybe we should wait until https://github.com/ceph/teuthology/pull/1510 merged and use copy of that method?

Reply:
@kshtsk : I see the problem now. `sh()` has changed in `teuthology.orchestra.remote.py` as per https://github.com/ceph/teuthology/commit/f05deb85e194cefa6bc8f5062c10866e08d812ec. So do you suggest a new fix of `sh()` in the ceph:master (based on the above commit) and then backport it?

Reply:
@joscollin yes, if we want to have the _sh_ method be the same.

Reply:
but I don't want to block this PR if someone want to merge it earlier

Reply:
Agreed, approving both the PRs.